point-of-care-poc/installation-and-configuration.md

```diff
@@ -2,5 +2,5 @@
 
 **Installation** Installation of point of care is same as installation of UgandaEMR. See [Release Notes 3.0.0](../release-notes/ugandaemr-300.md)
```

Comment:
@slubwama Installation point a link to the installation guide not the release notes